communication-service/utils/chatUsers.js

```diff
@@ -0,0 +1,36 @@
+const users = [];
```

Comment:
A suggestion to optimize runtime, can try using a Map to store users. Can work on this if got time